tests/unit/conftest.py

```diff
@@ -66,3 +67,12 @@ def plan_status() -> None:
     """Get an empty PlanStatus for every test case."""
     PlanStatus.error_messages = []
     PlanStatus.warning_messages = []
+
+
+@pytest.fixture
+def event_loop():
```

Comment:
Would it be useful to add a comment here that this is automatically used by pytest asyncio (or that it overrides the internal fixture provided already by pytest-asyncio - I'm not sure exactly how it works)?